cmd/snapshot-validation-webhook/main.go

```diff
@@ -20,7 +20,7 @@ import (
 	"flag"
 
 	webhook "github.com/kubernetes-csi/external-snapshotter/v3/pkg/validation-webhook"
-	"k8s.io/klog"
+	klog "k8s.io/klog/v2"
```

Comment:
nit: you don't need to rename the module on import. It's not wrong, just redundant. Did you add that manually or was that something that your editor did?

Reply:
Done